subprojects/packagefiles/protobuf-c/meson.build

```diff
@@ -1,20 +1,23 @@
-project('protobuf-c', 'c', 'cpp', version : '1.3.0', license : 'BSD-2')
+project('protobuf-c', 'c', 'cpp', version : '1.4.0', license : 'BSD-2')
 
-c = meson.get_compiler('c')
 cc = meson.get_compiler('cpp')
 
 libprotobuf_c_src = [
   'protobuf-c/protobuf-c.c',
 ]
 
-protobuf_deps = dependency('protobuf',
-                           fallback: ['protobuf', 'protobuf_dep'],
-                           version: '>= 3.0.0')
-libprotobuf_c = library('protobuf-c',
-                        libprotobuf_c_src,
-                        cpp_args: [],
-                        dependencies: protobuf_deps,
-                        include_directories: include_directories('protobuf-c'))
+protobuf_deps = dependency(
+  'protobuf',
+  fallback: ['protobuf', 'protobuf_dep'],
+  version: '>= 3.0.0'
+)
+
+libprotobuf_c = library(
+  'protobuf-c',
+  libprotobuf_c_src,
+  dependencies: protobuf_deps,
+  include_directories: include_directories('.')
```

Comment:
This changed from "protobuf-c" apparently? Note that "." is already added by default...